### PR TITLE
BACK-969: CI is now on Slack

### PR DESCRIPTION
**Pull request by jjayet-ledger**
## What is this about?

CircleCI sends Slack notifications in case of CI errors.

## Cute picture of lama
44337b67caa020912aaf83c6dc5fb2f0

### CODE REVIEW

**Comment by estrauser-ledger** [comment]
Dat llama is on fire !!

File: coins/bitcoin/interpreter/src/it/scala/co.ledger.lama.bitcoin.interpreter/OperationInterpreterIT.scala [hunk 1]
```diff
@@ -84,7 +84,7 @@ class OperationInterpreterIT extends AnyFlatSpecLike with Matchers with TestReso
           op.hash shouldBe insertTx.hash
           op.operationType shouldBe Sent
 
-          tx.fees shouldBe insertTx.fees
+          tx.fees shouldNot be(insertTx.fees)
```
Comment (estrauser-ledger) [inline coins/bitcoin/interpreter/src/it/scala/co.ledger.lama.bitcoin.interpreter/OperationInterpreterIT.scala:87]: ??? 
This should work it's very strange.
(btw, with 'not' you can write : `x should ne bot y`)

Comment (estrauser-ledger) [reply coins/bitcoin/interpreter/src/it/scala/co.ledger.lama.bitcoin.interpreter/OperationInterpreterIT.scala:87]: Oh I just read the commit message...

**Review by estrauser-ledger — APPROVED** [review]
love the llama bro :+1:

File: .circleci/config.yml [hunk 2]
```diff
@@ -98,6 +99,77 @@ jobs:
       - run:
           name: Run Bitcoin Interpreter unit tests
           command: << pipeline.parameters.sbt >> bitcoinInterpreter/test
+      - slack/notify:
```
Comment (nukky94) [inline .circleci/config.yml:102]: You can make 2 commands: `slack_notify_on_fail`, `slack_notify_on_pass`

Comment (estrauser-ledger) [reply .circleci/config.yml:102]: the image is not the same depending on the job that passed

Comment (nukky94) [reply .circleci/config.yml:102]: Commands support parameters

**Pull request merged**